src/components/ComposerQuill/composer.js

```diff
@@ -125,6 +125,12 @@ class Composer extends React.Component {
         placeholder,
       });
 
+      if (mentionListLabel) {
+        this.quill.getModule('mention').mentionContainer.setAttribute('aria-label', mentionListLabel);
```

Comment:
suggestion: Instead of getting the module every time it is needed, why not just store the module instance on the class (like what is done for quill), i.e.

```js
// In constructor
this.quillMention = undefined

// In componentDidMount
this.quillMention = this.quill.getModule('mention');

if (mentionListLabel) {
  this.quillMention.mentionContainer...
}
...

// in handleMentionOpen
this.quillMention.mentionContainer...
```

Hopefully you understand what I am getting at! Not a big deal since it isn't a huge performance hit, but might make it more readable.

Reply:
Only due to mild anxiety about the module changing for some reason, I'll change it